Flaky reorg e2e test

### Describe the feature

Fix the flaky test

```
    el.go:49:                   INFO [12-18|08:00:53.147] Target block reached                     chain=901 block=131b98..de224f:29
    reorg_test.go:173:          INFO [12-18|08:00:53.149] Reorged chain on divergence block number (prior the reorg) number=14 head=143097..4f79e6    parent=f6b41c..40d262
    reorg_test.go:174:          INFO [12-18|08:00:53.149] Reorged chain on divergence block number (after the reorg) number=14 head=aa138d..3c3a4a    parent=f6b41c..40d262
    assertions.go:369:             ERROR[12-18|08:00:53.229] "\n\tError Trace:\t/home/runner/work/op-reth/op-reth/crates/optimism/tests/proofs/utils/proof.go:121\n\t            \t\t\t\t/home/runner/work/op-reth/op-reth/crates/optimism/tests/proofs/reorg/reorg_test.go:180\n\tError:      \tNot equal: \n\t            \texpected: &eth.AccountResult{AccountProof:[]hexutil.Bytes{hexutil.Bytes{0xf9, 0x2, 0x11, 0xa0, 0x45, 0x63, 0xda, 0xc6, 0x90, 0xed, 0x20, 0x7a, 0x4a, 0x51, 0x91, 0xe2, 0x31, 0xb5, 0xc1, 0x99, 0x68, 0x90, 0x4d, 0x77, 0x94, 0xa4, 0xb, 0x28, 0x96, 0x64, 0x5e, 0x33, 0xac, 0x3a, 0xb6, 0xa4, 0xa0, 0xf6, 0xe9, 0x95, 0xee, 0x97, 0x64, 0x9d, 0xbf, 0xff, 0x15, 0xaa, 0x3b, 0x14, 0xdb, 0x85, 0xed, 0x90, 0x37, 0xf2, 0x11, 0x4, 0x46, 0xba, 0x20, 0x6d, 0x7a, 0x36, 0x25, 0xf1, 0xf2, 0xb5, 0x38, 0xa0, 0x56, 0xe6, 0xab, 0xa3, 0xe9, 0x5d, 0xbf, 0xe7, 0xe0, 0xbc, 0x4e, 0x4b, 0xea, 0xeb, 0xc4, 0xa8, 0xc9, 0x34, 0x83, 0x49, 0x3b, 0x2b, 0x50, 0xcb, 0xc7, 0x6e, 0x9c, 0x58, 0x99, 0xfa, 0x97, 0x81, 0xa0, 0x9f, 0x4b, 0xc7, 0x87, 0xde, 0x44, 0x75, 0xb8, 0x3e, 0x5b, 0x9e, 0xca, 0xe3, 0x6e, 0x9, 0x49, 0x4, 0x53, 0xf5, 0x5b, 0x12, 0xd1, 0x58, 0xdb, 0xa, 0x7e, 0x92, 0xf2, 0x6e, 0x71, 0x75, 0xaf, 0xa0, 0xb1, 0x7, 0x22, 0xbe, 0xa1, 0x52, 0x34, 0x64, 0xb2, 0xdc, 0x4a, 0xcc, 0xb0, 0x5d, 0xb3, 0xfa, 0x8c, 0x44, 0xe6, 0x93, 0xe1, 0x44, 0xe5, 0xf1, 0x14, 0xec, 0xd6, 0x8, 0xfa, 0xb7, 0x87, 0x68, 0xa0, 0xde, 0x7e, 0x40, 0x38, 0xf3, 0x82, 0x1f, 0xdf, 0x20, 0xd4, 0x18, 0xba, 0xa7, 0x7c, 0x1, 0x18, 0x2, 0x67, 0xa2, 0x19, 0xcf, 0x52, 0x56, 0x62, 0xe8, 0x18, 0x3b, 0xbd, 0x0, 0xf4, 0x16, 0x66, 0xa0, 0x9a, 0x9f, 0x79, 0xb7, 0x92, 0x87, 0x7b, 0xd3, 0xb5, 0x20, 0xcb, 0xa5, 0xe, 0xd7, 0xe7, 0x19, 0xc9, 0x6e, 0xca, 0x9, 0x28, 0x51, 0x31, 0x5a, 0x32, 0x86, 0x4a, 0x97, 0x5e, 0x9, 0x5f, 0xeb, 0xa0, 0x73, 0x4f, 0x26, 0x82, 0x99, 0x1a, 0xad, 0xe, 0x1f, 0x56, 0xd5, 0x1d, 0x12, 0xe0, 0x90, 0x68, 0xb7, 0x95, 0xd6, 0x35, 0xe6, 0x30, 0xc7, 0xa4, 0x29, 0x8, 0xdd, 0xaa, 0x23, 0x6e, 0xc2, 0x5b, 0xa0, 0x80, 0x2c, 0xd3, 0x93, 0xe, 0x68, 0x7e, 0x26, 0x38, 0x6c, 0x2c, 0xe6, 0xf9, 0x3c, 0x73, 0xe5, 0xd5, 0x9b, 0xd9, 0xd, 0x39, 0x2, 0x55, 0xbb, 0xd4, 0x75, 0xaa, 0xf, 0x1c, 0x84, 0x4d, 0x7d, 0xa0, 0x28, 0xeb, 0x2f, 0x59, 0x19, 0x56, 0x77, 0xc5, 0x4d, 0xbc, 0x43, 0x7, 0x3c, 0x39, 0x6, 0xc0, 0x6a, 0x21, 0xd, 0x7a, 0xb8, 0x84, 0x3, 0x54, 0x52, 0xe0, 0x5, 0x61, 0x2e, 0x82, 0x1b, 0x71, 0xa0, 0x48, 0x9d, 0x48, 0x84, 0x4a, 0xd, 0xdb, 0xb1, 0x9, 0xb3, 0xdb, 0xdc, 0xd6, 0xbe, 0x7d, 0x8b, 0x6f, 0x12, 0xb4, 0xa0, 0x6d, 0x6a, 0x34, 0xde, 0xb7, 0xfb, 0xbd, 0x99, 0x59, 0xc0, 0x34, 0x4e, 0xa0, 0x1e, 0xaf, 0x9c, 0xca, 0x1a, 0xfe, 0x64, 0x2c, 0xe9, 0x5, 0x1e, 0x24, 0x73, 0x4a, 0xb1, 0x9d, 0x4f, 0x46, 0xf0, 0x79, 0xb4, 0x0, 0xc0, 0xc, 0xaa, 0xd5, 0x5, 0xb9, 0x73, 0xaf, 0xcd, 0x55, 0xa0, 0x8b, 0x8f, 0x14, 0xfd, 0xed, 0x6a, 0x50, 0x98, 0x15, 0x16, 0x89, 0x2, 0x65, 0x50, 0x11, 0xa3, 0x58, 0x9b, 0x9e, 0x25, 0xcd, 0x2, 0xe8, 0xdc, 0x40, 0x7c, 0xd0, 0x53, 0xfd, 0x27, 0xb7, 0x92, 0xa0, 0xf0, 0x4a, 0xa3, 0xa7, 0xad, 0x13, 0x88, 0x5f, 0x3a, 0x64, 0x10, 0x5c, 0x73, 0xa, 0xbc, 0x4b, 0xe, 0x16, 0x6e, 0xc8, 0x30, 0xa8, 0x88, 0x2d, 0xcf, 0x5e, 0xe5, 0x3e, 0x9f, 0x40, 0xeb, 0xfd, 0xa0, 0xbd, 0x4a, 0x30, 0xab, 0xde, 0xaa, 0xda, 0x1, 0x64, 0x5b, 0x10, 0xea, 0xce, 0x3b, 0xa0, 0x43, 0xdd, 0xd2, 0x81, 0x54, 0xde, 0x3b, 0xc0, 0xb8, 0x7f, 0x32, 0xf, 0x28, 0xde, 0xf7, 0xfa, 0xbe, 0xa0, 0xe6, 0xc, 0x9e, 0x2a, 0x1f, 0xf9, 0x8, 0x37, 0xc8, 0xae, 0x8f, 0xd6, 0xdf, 0x9f, 0x76, 0xfe, 0xcd, 0x10, 0x1e, 0xaa, 0x8c, 0x5a, 0x34, 0xbe, 0x72, 0xcb, 0x5b, 0x4, 0xe4, 0x15, 0xd4, 0xb7, 0x80}, hexutil.Bytes{0xf9, 0x2, 0x11, 0xa0, 0xa2, 0x97, 0x25, 0xa1, 0x66, 0xe5, 0x5b, 0x94, 0x5f, 0x9d, 0xbf, 0x10, 0xd1, 0xc5, 0x57, 0x96, 0x48, 0xb3, 0x45, 0x8f, 0x3e, 0x63, 0xb4, 0x62, 0x13, 0x6d, 0x1e, 0x5e, 0xc4, 0x83, 0x95, 0x19, 0xa0, 0x50, 0xb6, 0xe3, 0x5c, 0x60, 0x68, 0x4d, 0x16, 0x5, 0x1c, 0x39, 0x44, 0x61, 0x1d, 0x2e, 0x3b, 0xae, 0x8f, 0xc2, 0xed, 0x5, 0x8, 0xce, 0x1b, 0x62, 0xc1, 0x1e, 0xa3, 0x7a, 0x39, 0xdd, 0x5a, 0xa0, 0x2, 0xbf, 0xa0, 0x98, 0x5d, 0xa2, 0xfd, 0xaa, 0xd9, 0xaa, 0xbc, 0xd1, 0xbc, 0xc6, 0xb6, 0x2, 0x71, 0x12, 0x2a, 0xa6, 0x37, 0xdb, 0x57, 0xca, 0xee, 0xcb, 0x5, 0x1c, 0xfb, 0xfb, 0x6e, 0x21, 0xa0, 0x9a, 0xe5, 0x16, 0x22, 0xd4, 0x39, 0xe2, 0xa1, 0x1, 0x57, 0xdb, 0x69, 0xf1, 0x5f, 0x7, 0x6e, 0x74, 0x69, 0xd7, 0xaa, 0x26, 0xc8, 0xdd, 0x41, 0xb4, 0xcb, 0x1a, 0xed, 0x26, 0xa5, 0xb7, 0xfb, 0xa0, 0xd2, 0x7e, 0xfe, 0xba, 0x79, 0x87, 0xd2, 0x2c, 0x3b, 0x81, 0x4a, 0x50, 0xfd, 0x50, 0xd5, 0x4c, 0x8c, 0xff, 0x58, 0x53, 0xa9, 0xde, 0xfd, 0x31, 0x3, 0x53, 0xd5, 0xd1, 0x48, 0x53, 0xa9, 0x30, 0xa0, 0xb0, 0x6d, 0x45, 0x93, 0x2a, 0x31, 0x80, 0x2c, 0x21, 0x9f, 0x10, 0x9, 0xdc, 0xe9, 0xe7, 0x99, 0x69, 0x32, 0xf4, 0x86, 0x44, 0x8d, 0x4a, 0x94, 0x25, 0x9d, 0xc2, 0x80, 0x4d, 0xb6, 0x5b, 0x8a, 0xa0, 0xc1, 0x72, 0x18, 0xf3, 0xd9, 0x4f, 0xb9, 0x59, 0x9e, 0xda, 0xd9, 0x11, 0xd7, 0x17, 0x2c, 0xdc, 0x38, 0xb9, 0x48, 0x76, 0x7, 0x11, 0xf6, 0xe0, 0x76, 0x50, 0x45, 0x83, 0x21, 0xba, 0xea, 0xaf, 0xa0, 0xcb, 0xfe, 0x86, 0x9, 0x67, 0xd9, 0x95, 0xeb, 0xb9, 0xea, 0xe0, 0x63, 0xa7, 0x98, 0x3e, 0x98, 0x7c, 0xb, 0xc2, 0x42, 0xca, 0x14, 0x99, 0x18, 0xdb, 0xc3, 0xe3, 0x72, 0x78, 0x45, 0x9e, 0xfc, 0xa0, 0xe6, 0x97, 0x26, 0x46, 0x6f, 0x9e, 0x9f, 0xba, 0x6, 0x46, 0x8, 0xc, 0x5a, 0x45, 0x8a, 0x5e, 0xe2, 0xec, 0x2, 0xcd, 0x76, 0x95, 0x3b, 0x6a, 0xdd, 0x79, 0x74, 0x65, 0x73, 0x96, 0xea, 0xa9, 0xa0, 0xf3, 0xd9, 0x97, 0x3d, 0x10, 0xe4, 0xa3, 0xf1, 0x31, 0xb3, 0xea, 0x23, 0x2c, 0x9a, 0x36, 0xc9, 0xa1, 0x2d, 0xf9, 0x92, 0xdc, 0xf8, 0x94, 0x68, 0xc1, 0xb2, 0x27, 0xda, 0x31, 0x8c, 0xf3, 0xff, 0xa0, 0xce, 0x2d, 0x9e, 0xe3, 0x7b, 0x6f, 0x68, 0xe1, 0x48, 0x5, 0x8d, 0x18, 0x64, 0x25, 0x38, 0x6c, 0x22, 0x9b, 0x20, 0xb5, 0xa5, 0x20, 0x60, 0xd0, 0xc5, 0xc9, 0x14, 0x60, 0x19, 0xe0, 0x91, 0x92, 0xa0, 0x99, 0xd9, 0x30, 0xc8, 0xb3, 0xea, 0x5e, 0x36, 0x3, 0x4b, 0x16, 0xb1, 0x1f, 0xf7, 0xd0, 0x8c, 0x92, 0x22, 0x4d, 0x3c, 0x7e, 0xbe, 0xcc, 0x71, 0x13, 0x80, 0x26, 0xf4, 0xa3, 0x27, 0x60, 0x83, 0xa0, 0x8c, 0xcf, 0x5d, 0x52, 0x4a, 0x2f, 0x7f, 0x13, 0x2c, 0x35, 0x30, 0x99, 0xcb, 0x7d, 0x8f, 0x6c, 0x96, 0xc7, 0x7c, 0x84, 0xb, 0xed, 0xf4, 0xf1, 0xb9, 0xae, 0xbc, 0x79, 0x8e, 0x86, 0x5e, 0x3c, 0xa0, 0xc, 0x59, 0x81, 0x17, 0x27, 0x93, 0x12, 0x24, 0x17, 0xf2, 0xdc, 0xfb, 0x46, 0x8a, 0xa0, 0xef, 0x2, 0xc3, 0xaf, 0xef, 0xe, 0xfe, 0x8d, 0xcd, 0x5f, 0x14, 0x1e, 0x79, 0x26, 0xe6, 0x42, 0xd2, 0xa0, 0x2d, 0xe2, 0xa0, 0xcc, 0x50, 0xa5, 0x34, 0x34, 0x4f, 0xb7, 0xe1, 0x7f, 0xec, 0x6e, 0x6c, 0x7e, 0x54, 0xd0, 0x7a, 0xe4, 0x97, 0x66, 0x9, 0x7c, 0x25, 0x38, 0x90, 0x40, 0x67, 0x46, 0xa1, 0x37, 0xa0, 0x24, 0x6d, 0x83, 0x4a, 0xdd, 0x91, 0xa, 0xa5, 0x6c, 0x7, 0x89, 0x6d, 0xaa, 0x52, 0x7b, 0x20, 0x7f, 0xc, 0xe3, 0x2f, 0xfc, 0xa9, 0xb1, 0x3b, 0xa9, 0x2e, 0x86, 0x3d, 0xd3, 0x76, 0x1a, 0x2a, 0x80}, hexutil.Bytes{0xf9, 0x1, 0x31, 0x80, 0x80, 0xa0, 0x4a, 0x29, 0xc8, 0xc0, 0xc0, 0xee, 0xe2, 0xc, 0x55, 0xe1, 0xa, 0x88, 0x6c, 0xe8, 0xf2, 0x74, 0x8e, 0x9f, 0x0, 0xab, 0x70, 0xc0, 0xe2, 0x59, 0xc, 0xb8, 0x4a, 0x1e, 0x99, 0xdc, 0xa8, 0x83, 0xa0, 0xf0, 0xda, 0x9, 0xd1, 0xad, 0x3f, 0x53, 0x29, 0xc0, 0x17, 0xc5, 0xb0, 0x83, 0x24, 0xc6, 0xe1, 0xc1, 0x98, 0x7, 0xaa, 0x55, 0x66, 0x42, 0x4c, 0x30, 0x57, 0x21, 0x71, 0xb1, 0x25, 0x93, 0xfd, 0x80, 0xa0, 0x50, 0x45, 0x6a, 0x8e, 0x5c, 0x46, 0xd4, 0xb8, 0x3a, 0x36, 0xcf, 0x3d, 0xa0, 0x69, 0x41, 0xf8, 0xaa, 0x64, 0x7, 0xe0, 0x34, 0xf9, 0x6, 0x12, 0x1d, 0x1f, 0x3c, 0xf4, 0x4a, 0x4a, 0x46, 0x44, 0x80, 0xa0, 0xe0, 0x47, 0x2d, 0x33, 0xbe, 0x3f, 0xbd, 0x30, 0x4d, 0x25, 0x66, 0xb3, 0xc8, 0x1c, 0xd4, 0xa5, 0xf5, 0x3b, 0xaa, 0xb8, 0x9e, 0xc1, 0x75, 0x94, 0x27, 0x86, 0xf, 0x55, 0x89, 0xf1, 0x65, 0xe2, 0xa0, 0x9a, 0x84, 0x67, 0xa3, 0x97, 0xe0, 0xdd, 0x46, 0x26, 0x6f, 0xcd, 0xad, 0xd0, 0x34, 0xba, 0xad, 0x80, 0x2b, 0xa1, 0x79, 0xd8, 0xa1, 0x83, 0xf4, 0x62, 0x37, 0xab, 0xa4, 0xd0, 0x5d, 0x1a, 0xf0, 0xa0, 0xe2, 0x9b, 0xd, 0x6b, 0x80, 0x4c, 0x1, 0xf5, 0x26, 0x97, 0x89, 0xe6, 0x68, 0xcc, 0x4a, 0x57, 0xc, 0x4c, 0x5f, 0x24, 0x24, 0xb7, 0x1d, 0x81, 0xdd, 0x10, 0xa1, 0x22, 0xfa, 0x40, 0xc1, 0x58, 0x80, 0xa0, 0xf4, 0x2, 0x73, 0x50, 0xa0, 0x58, 0x2a, 0x91, 0x1f, 0x4f, 0x19, 0xa6, 0x96, 0xde, 0x1a, 0xe9, 0xf6, 0xd8, 0xfc, 0xcb, 0xea, 0xcf, 0x8e, 0xdd, 0x90, 0xae, 0x38, 0x70, 0x6d, 0x1d, 0x99, 0xba, 0x80, 0x80, 0xa0, 0x90, 0x29, 0x42, 0x78, 0x35, 0x78, 0x4b, 0xb, 0x64, 0xd, 0xf4, 0x13, 0x46, 0x73, 0xa6, 0xe0, 0x9e, 0xd5, 0x84, 0x71, 0xae, 0xd2, 0x7d, 0xc2, 0xc5, 0xeb, 0x33, 0x26, 0x58, 0xcb, 0x45, 0x64, 0xa0, 0xc4, 0x48, 0x9d, 0x25, 0x98, 0xb2, 0xcd, 0x6a, 0xb3, 0xa6, 0xfd, 0x4a, 0xa4, 0xc8, 0xf0, 0x53, 0xc0, 0x33, 0x6d, 0xfa, 0xe4, 0x8f, 0xc6, 0xe1, 0x6e, 0xe8, 0xfb, 0x21, 0x31, 0xb7, 0x9, 0xc1, 0x80}}, Address:0x64aFE71F8f4Bf7746B444802E5cDC58F4a5627Fe, Balance:(*hexutil.Big)(0xc0007b8d80), CodeHash:0x0000000000000000000000000000000000000000000000000000000000000000, Nonce:0x0, StorageHash:0x0000000000000000000000000000000000000000000000000000000000000000, StorageProof:[]eth.StorageProofEntry{eth.StorageProofEntry{Key:eth.StorageKey{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0}, Value:hexutil.Big{neg:false, abs:big.nat{}}, Proof:[]hexutil.Bytes{}}}}\n\t            \tactual  : &eth.AccountResult{AccountProof:[]hexutil.Bytes{hexutil.Bytes{0xf9, 0x2, 0x11, 0xa0, 0x45, 0x63, 0xda, 0xc6, 0x90, 0xed, 0x20, 0x7a, 0x4a, 0x51, 0x91, 0xe2, 0x31, 0xb5, 0xc1, 0x99, 0x68, 0x90, 0x4d, 0x77, 0x94, 0xa4, 0xb, 0x28, 0x96, 0x64, 0x5e, 0x33, 0xac, 0x3a, 0xb6, 0xa4, 0xa0, 0xf6, 0xe9, 0x95, 0xee, 0x97, 0x64, 0x9d, 0xbf, 0xff, 0x15, 0xaa, 0x3b, 0x14, 0xdb, 0x85, 0xed, 0x90, 0x37, 0xf2, 0x11, 0x4, 0x46, 0xba, 0x20, 0x6d, 0x7a, 0x36, 0x25, 0xf1, 0xf2, 0xb5, 0x38, 0xa0, 0x56, 0xe6, 0xab, 0xa3, 0xe9, 0x5d, 0xbf, 0xe7, 0xe0, 0xbc, 0x4e, 0x4b, 0xea, 0xeb, 0xc4, 0xa8, 0xc9, 0x34, 0x83, 0x49, 0x3b, 0x2b, 0x50, 0xcb, 0xc7, 0x6e, 0x9c, 0x58, 0x99, 0xfa, 0x97, 0x81, 0xa0, 0x9f, 0x4b, 0xc7, 0x87, 0xde, 0x44, 0x75, 0xb8, 0x3e, 0x5b, 0x9e, 0xca, 0xe3, 0x6e, 0x9, 0x49, 0x4, 0x53, 0xf5, 0x5b, 0x12, 0xd1, 0x58, 0xdb, 0xa, 0x7e, 0x92, 0xf2, 0x6e, 0x71, 0x75, 0xaf, 0xa0, 0xb1, 0x7, 0x22, 0xbe, 0xa1, 0x52, 0x34, 0x64, 0xb2, 0xdc, 0x4a, 0xcc, 0xb0, 0x5d, 0xb3, 0xfa, 0x8c, 0x44, 0xe6, 0x93, 0xe1, 0x44, 0xe5, 0xf1, 0x14, 0xec, 0xd6, 0x8, 0xfa, 0xb7, 0x87, 0x68, 0xa0, 0xde, 0x7e, 0x40, 0x38, 0xf3, 0x82, 0x1f, 0xdf, 0x20, 0xd4, 0x18, 0xba, 0xa7, 0x7c, 0x1, 0x18, 0x2, 0x67, 0xa2, 0x19, 0xcf, 0x52, 0x56, 0x62, 0xe8, 0x18, 0x3b, 0xbd, 0x0, 0xf4, 0x16, 0x66, 0xa0, 0x9a, 0x9f, 0x79, 0xb7, 0x92, 0x87, 0x7b, 0xd3, 0xb5, 0x20, 0xcb, 0xa5, 0xe, 0xd7, 0xe7, 0x19, 0xc9, 0x6e, 0xca, 0x9, 0x28, 0x51, 0x31, 0x5a, 0x32, 0x86, 0x4a, 0x97, 0x5e, 0x9, 0x5f, 0xeb, 0xa0, 0x73, 0x4f, 0x26, 0x82, 0x99, 0x1a, 0xad, 0xe, 0x1f, 0x56, 0xd5, 0x1d, 0x12, 0xe0, 0x90, 0x68, 0xb7, 0x95, 0xd6, 0x35, 0xe6, 0x30, 0xc7, 0xa4, 0x29, 0x8, 0xdd, 0xaa, 0x23, 0x6e, 0xc2, 0x5b, 0xa0, 0x80, 0x2c, 0xd3, 0x93, 0xe, 0x68, 0x7e, 0x26, 0x38, 0x6c, 0x2c, 0xe6, 0xf9, 0x3c, 0x73, 0xe5, 0xd5, 0x9b, 0xd9, 0xd, 0x39, 0x2, 0x55, 0xbb, 0xd4, 0x75, 0xaa, 0xf, 0x1c, 0x84, 0x4d, 0x7d, 0xa0, 0x28, 0xeb, 0x2f, 0x59, 0x19, 0x56, 0x77, 0xc5, 0x4d, 0xbc, 0x43, 0x7, 0x3c, 0x39, 0x6, 0xc0, 0x6a, 0x21, 0xd, 0x7a, 0xb8, 0x84, 0x3, 0x54, 0x52, 0xe0, 0x5, 0x61, 0x2e, 0x82, 0x1b, 0x71, 0xa0, 0x48, 0x9d, 0x48, 0x84, 0x4a, 0xd, 0xdb, 0xb1, 0x9, 0xb3, 0xdb, 0xdc, 0xd6, 0xbe, 0x7d, 0x8b, 0x6f, 0x12, 0xb4, 0xa0, 0x6d, 0x6a, 0x34, 0xde, 0xb7, 0xfb, 0xbd, 0x99, 0x59, 0xc0, 0x34, 0x4e, 0xa0, 0x1e, 0xaf, 0x9c, 0xca, 0x1a, 0xfe, 0x64, 0x2c, 0xe9, 0x5, 0x1e, 0x24, 0x73, 0x4a, 0xb1, 0x9d, 0x4f, 0x46, 0xf0, 0x79, 0xb4, 0x0, 0xc0, 0xc, 0xaa, 0xd5, 0x5, 0xb9, 0x73, 0xaf, 0xcd, 0x55, 0xa0, 0x8b, 0x8f, 0x14, 0xfd, 0xed, 0x6a, 0x50, 0x98, 0x15, 0x16, 0x89, 0x2, 0x65, 0x50, 0x11, 0xa3, 0x58, 0x9b, 0x9e, 0x25, 0xcd, 0x2, 0xe8, 0xdc, 0x40, 0x7c, 0xd0, 0x53, 0xfd, 0x27, 0xb7, 0x92, 0xa0, 0xf0, 0x4a, 0xa3, 0xa7, 0xad, 0x13, 0x88, 0x5f, 0x3a, 0x64, 0x10, 0x5c, 0x73, 0xa, 0xbc, 0x4b, 0xe, 0x16, 0x6e, 0xc8, 0x30, 0xa8, 0x88, 0x2d, 0xcf, 0x5e, 0xe5, 0x3e, 0x9f, 0x40, 0xeb, 0xfd, 0xa0, 0xbd, 0x4a, 0x30, 0xab, 0xde, 0xaa, 0xda, 0x1, 0x64, 0x5b, 0x10, 0xea, 0xce, 0x3b, 0xa0, 0x43, 0xdd, 0xd2, 0x81, 0x54, 0xde, 0x3b, 0xc0, 0xb8, 0x7f, 0x32, 0xf, 0x28, 0xde, 0xf7, 0xfa, 0xbe, 0xa0, 0xe6, 0xc, 0x9e, 0x2a, 0x1f, 0xf9, 0x8, 0x37, 0xc8, 0xae, 0x8f, 0xd6, 0xdf, 0x9f, 0x76, 0xfe, 0xcd, 0x10, 0x1e, 0xaa, 0x8c, 0x5a, 0x34, 0xbe, 0x72, 0xcb, 0x5b, 0x4, 0xe4, 0x15, 0xd4, 0xb7, 0x80}, hexutil.Bytes{0xf9, 0x2, 0x11, 0xa0, 0xa2, 0x97, 0x25, 0xa1, 0x66, 0xe5, 0x5b, 0x94, 0x5f, 0x9d, 0xbf, 0x10, 0xd1, 0xc5, 0x57, 0x96, 0x48, 0xb3, 0x45, 0x8f, 0x3e, 0x63, 0xb4, 0x62, 0x13, 0x6d, 0x1e, 0x5e, 0xc4, 0x83, 0x95, 0x19, 0xa0, 0x50, 0xb6, 0xe3, 0x5c, 0x60, 0x68, 0x4d, 0x16, 0x5, 0x1c, 0x39, 0x44, 0x61, 0x1d, 0x2e, 0x3b, 0xae, 0x8f, 0xc2, 0xed, 0x5, 0x8, 0xce, 0x1b, 0x62, 0xc1, 0x1e, 0xa3, 0x7a, 0x39, 0xdd, 0x5a, 0xa0, 0x2, 0xbf, 0xa0, 0x98, 0x5d, 0xa2, 0xfd, 0xaa, 0xd9, 0xaa, 0xbc, 0xd1, 0xbc, 0xc6, 0xb6, 0x2, 0x71, 0x12, 0x2a, 0xa6, 0x37, 0xdb, 0x57, 0xca, 0xee, 0xcb, 0x5, 0x1c, 0xfb, 0xfb, 0x6e, 0x21, 0xa0, 0x9a, 0xe5, 0x16, 0x22, 0xd4, 0x39, 0xe2, 0xa1, 0x1, 0x57, 0xdb, 0x69, 0xf1, 0x5f, 0x7, 0x6e, 0x74, 0x69, 0xd7, 0xaa, 0x26, 0xc8, 0xdd, 0x41, 0xb4, 0xcb, 0x1a, 0xed, 0x26, 0xa5, 0xb7, 0xfb, 0xa0, 0xd2, 0x7e, 0xfe, 0xba, 0x79, 0x87, 0xd2, 0x2c, 0x3b, 0x81, 0x4a, 0x50, 0xfd, 0x50, 0xd5, 0x4c, 0x8c, 0xff, 0x58, 0x53, 0xa9, 0xde, 0xfd, 0x31, 0x3, 0x53, 0xd5, 0xd1, 0x48, 0x53, 0xa9, 0x30, 0xa0, 0xb0, 0x6d, 0x45, 0x93, 0x2a, 0x31, 0x80, 0x2c, 0x21, 0x9f, 0x10, 0x9, 0xdc, 0xe9, 0xe7, 0x99, 0x69, 0x32, 0xf4, 0x86, 0x44, 0x8d, 0x4a, 0x94, 0x25, 0x9d, 0xc2, 0x80, 0x4d, 0xb6, 0x5b, 0x8a, 0xa0, 0xc1, 0x72, 0x18, 0xf3, 0xd9, 0x4f, 0xb9, 0x59, 0x9e, 0xda, 0xd9, 0x11, 0xd7, 0x17, 0x2c, 0xdc, 0x38, 0xb9, 0x48, 0x76, 0x7, 0x11, 0xf6, 0xe0, 0x76, 0x50, 0x45, 0x83, 0x21, 0xba, 0xea, 0xaf, 0xa0, 0xcb, 0xfe, 0x86, 0x9, 0x67, 0xd9, 0x95, 0xeb, 0xb9, 0xea, 0xe0, 0x63, 0xa7, 0x98, 0x3e, 0x98, 0x7c, 0xb, 0xc2, 0x42, 0xca, 0x14, 0x99, 0x18, 0xdb, 0xc3, 0xe3, 0x72, 0x78, 0x45, 0x9e, 0xfc, 0xa0, 0xe6, 0x97, 0x26, 0x46, 0x6f, 0x9e, 0x9f, 0xba, 0x6, 0x46, 0x8, 0xc, 0x5a, 0x45, 0x8a, 0x5e, 0xe2, 0xec, 0x2, 0xcd, 0x76, 0x95, 0x3b, 0x6a, 0xdd, 0x79, 0x74, 0x65, 0x73, 0x96, 0xea, 0xa9, 0xa0, 0xf3, 0xd9, 0x97, 0x3d, 0x10, 0xe4, 0xa3, 0xf1, 0x31, 0xb3, 0xea, 0x23, 0x2c, 0x9a, 0x36, 0xc9, 0xa1, 0x2d, 0xf9, 0x92, 0xdc, 0xf8, 0x94, 0x68, 0xc1, 0xb2, 0x27, 0xda, 0x31, 0x8c, 0xf3, 0xff, 0xa0, 0xce, 0x2d, 0x9e, 0xe3, 0x7b, 0x6f, 0x68, 0xe1, 0x48, 0x5, 0x8d, 0x18, 0x64, 0x25, 0x38, 0x6c, 0x22, 0x9b, 0x20, 0xb5, 0xa5, 0x20, 0x60, 0xd0, 0xc5, 0xc9, 0x14, 0x60, 0x19, 0xe0, 0x91, 0x92, 0xa0, 0x99, 0xd9, 0x30, 0xc8, 0xb3, 0xea, 0x5e, 0x36, 0x3, 0x4b, 0x16, 0xb1, 0x1f, 0xf7, 0xd0, 0x8c, 0x92, 0x22, 0x4d, 0x3c, 0x7e, 0xbe, 0xcc, 0x71, 0x13, 0x80, 0x26, 0xf4, 0xa3, 0x27, 0x60, 0x83, 0xa0, 0x8c, 0xcf, 0x5d, 0x52, 0x4a, 0x2f, 0x7f, 0x13, 0x2c, 0x35, 0x30, 0x99, 0xcb, 0x7d, 0x8f, 0x6c, 0x96, 0xc7, 0x7c, 0x84, 0xb, 0xed, 0xf4, 0xf1, 0xb9, 0xae, 0xbc, 0x79, 0x8e, 0x86, 0x5e, 0x3c, 0xa0, 0xc, 0x59, 0x81, 0x17, 0x27, 0x93, 0x12, 0x24, 0x17, 0xf2, 0xdc, 0xfb, 0x46, 0x8a, 0xa0, 0xef, 0x2, 0xc3, 0xaf, 0xef, 0xe, 0xfe, 0x8d, 0xcd, 0x5f, 0x14, 0x1e, 0x79, 0x26, 0xe6, 0x42, 0xd2, 0xa0, 0x2d, 0xe2, 0xa0, 0xcc, 0x50, 0xa5, 0x34, 0x34, 0x4f, 0xb7, 0xe1, 0x7f, 0xec, 0x6e, 0x6c, 0x7e, 0x54, 0xd0, 0x7a, 0xe4, 0x97, 0x66, 0x9, 0x7c, 0x25, 0x38, 0x90, 0x40, 0x67, 0x46, 0xa1, 0x37, 0xa0, 0x24, 0x6d, 0x83, 0x4a, 0xdd, 0x91, 0xa, 0xa5, 0x6c, 0x7, 0x89, 0x6d, 0xaa, 0x52, 0x7b, 0x20, 0x7f, 0xc, 0xe3, 0x2f, 0xfc, 0xa9, 0xb1, 0x3b, 0xa9, 0x2e, 0x86, 0x3d, 0xd3, 0x76, 0x1a, 0x2a, 0x80}, hexutil.Bytes{0xf9, 0x1, 0x31, 0x80, 0x80, 0xa0, 0x4a, 0x29, 0xc8, 0xc0, 0xc0, 0xee, 0xe2, 0xc, 0x55, 0xe1, 0xa, 0x88, 0x6c, 0xe8, 0xf2, 0x74, 0x8e, 0x9f, 0x0, 0xab, 0x70, 0xc0, 0xe2, 0x59, 0xc, 0xb8, 0x4a, 0x1e, 0x99, 0xdc, 0xa8, 0x83, 0xa0, 0xf0, 0xda, 0x9, 0xd1, 0xad, 0x3f, 0x53, 0x29, 0xc0, 0x17, 0xc5, 0xb0, 0x83, 0x24, 0xc6, 0xe1, 0xc1, 0x98, 0x7, 0xaa, 0x55, 0x66, 0x42, 0x4c, 0x30, 0x57, 0x21, 0x71, 0xb1, 0x25, 0x93, 0xfd, 0x80, 0xa0, 0x50, 0x45, 0x6a, 0x8e, 0x5c, 0x46, 0xd4, 0xb8, 0x3a, 0x36, 0xcf, 0x3d, 0xa0, 0x69, 0x41, 0xf8, 0xaa, 0x64, 0x7, 0xe0, 0x34, 0xf9, 0x6, 0x12, 0x1d, 0x1f, 0x3c, 0xf4, 0x4a, 0x4a, 0x46, 0x44, 0x80, 0xa0, 0xe0, 0x47, 0x2d, 0x33, 0xbe, 0x3f, 0xbd, 0x30, 0x4d, 0x25, 0x66, 0xb3, 0xc8, 0x1c, 0xd4, 0xa5, 0xf5, 0x3b, 0xaa, 0xb8, 0x9e, 0xc1, 0x75, 0x94, 0x27, 0x86, 0xf, 0x55, 0x89, 0xf1, 0x65, 0xe2, 0xa0, 0x9a, 0x84, 0x67, 0xa3, 0x97, 0xe0, 0xdd, 0x46, 0x26, 0x6f, 0xcd, 0xad, 0xd0, 0x34, 0xba, 0xad, 0x80, 0x2b, 0xa1, 0x79, 0xd8, 0xa1, 0x83, 0xf4, 0x62, 0x37, 0xab, 0xa4, 0xd0, 0x5d, 0x1a, 0xf0, 0xa0, 0xe2, 0x9b, 0xd, 0x6b, 0x80, 0x4c, 0x1, 0xf5, 0x26, 0x97, 0x89, 0xe6, 0x68, 0xcc, 0x4a, 0x57, 0xc, 0x4c, 0x5f, 0x24, 0x24, 0xb7, 0x1d, 0x81, 0xdd, 0x10, 0xa1, 0x22, 0xfa, 0x40, 0xc1, 0x58, 0x80, 0xa0, 0xf4, 0x2, 0x73, 0x50, 0xa0, 0x58, 0x2a, 0x91, 0x1f, 0x4f, 0x19, 0xa6, 0x96, 0xde, 0x1a, 0xe9, 0xf6, 0xd8, 0xfc, 0xcb, 0xea, 0xcf, 0x8e, 0xdd, 0x90, 0xae, 0x38, 0x70, 0x6d, 0x1d, 0x99, 0xba, 0x80, 0x80, 0xa0, 0x90, 0x29, 0x42, 0x78, 0x35, 0x78, 0x4b, 0xb, 0x64, 0xd, 0xf4, 0x13, 0x46, 0x73, 0xa6, 0xe0, 0x9e, 0xd5, 0x84, 0x71, 0xae, 0xd2, 0x7d, 0xc2, 0xc5, 0xeb, 0x33, 0x26, 0x58, 0xcb, 0x45, 0x64, 0xa0, 0xc4, 0x48, 0x9d, 0x25, 0x98, 0xb2, 0xcd, 0x6a, 0xb3, 0xa6, 0xfd, 0x4a, 0xa4, 0xc8, 0xf0, 0x53, 0xc0, 0x33, 0x6d, 0xfa, 0xe4, 0x8f, 0xc6, 0xe1, 0x6e, 0xe8, 0xfb, 0x21, 0x31, 0xb7, 0x9, 0xc1, 0x80}}, Address:0x64aFE71F8f4Bf7746B444802E5cDC58F4a5627Fe, Balance:(*hexutil.Big)(0xc0007b90c0), CodeHash:0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470, Nonce:0x0, StorageHash:0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421, StorageProof:[]eth.StorageProofEntry{eth.StorageProofEntry{Key:eth.StorageKey{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0}, Value:hexutil.Big{neg:false, abs:big.nat{}}, Proof:[]hexutil.Bytes{}}}}\n\t            \t\n\t            \tDiff:\n\t            \t--- Expected\n\t            \t+++ Actual\n\t            \t@@ -107,4 +107,4 @@\n\t            \t  CodeHash: (common.Hash) (len=32) {\n\t            \t-  00000000  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|\n\t            \t-  00000010  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|\n\t            \t+  00000000  c5 d2 46 01 86 f7 23 3c  92 7e 7d b2 dc c7 03 c0  |..F...#<.~}.....|\n\t            \t+  00000010  e5 00 b6 53 ca 82 27 3b  7b fa d8 04 5d 85 a4 70  |...S..';{...]..p|\n\t            \t  },\n\t            \t@@ -112,4 +112,4 @@\n\t            \t  StorageHash: (common.Hash) (len=32) {\n\t            \t-  00000000  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|\n\t            \t-  00000010  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|\n\t            \t+  00000000  56 e8 1f 17 1b cc 55 a6  ff 83 45 e6 92 c0 f8 6e  |V.....U...E....n|\n\t            \t+  00000010  5b 48 e0 1b 99 6c ad c0  01 62 2f b5 e3 63 b4 21  |[H...l...b/..c.!|\n\t            \t  },\n\tTest:       \tTestReorgUsingAccountProof\n\tMessages:   \tgeth and reth proofs should match\n"
--- FAIL: TestReorgUsingAccountProof (64.67s)
FAIL
```

### Additional context

_No response_

--- a/crates/optimism/tests/proofs/reorg/reorg_test.go
+++ b/crates/optimism/tests/proofs/reorg/reorg_test.go
@@ -170,13 +170,13 @@ func TestReorgUsingAccountProof(gt *testing.T) {
 
 	// verify that the L2A validator has reorged and reached the latest block
 	err := wait.For(t.Ctx(), 2*time.Second, func() (bool, error) {
-		block_ref, err := sys.L2ELValidatorNode().Escape().EthClient().BlockRefByNumber(ctx, latestBlock.Number)
+		blockRef, err := sys.L2ELValidatorNode().Escape().EthClient().BlockRefByNumber(ctx, latestBlock.Number)
 		if err != nil {
 			// this could happen if the validator is still syncing after reorg
 			l.Warn("Error fetching block reference from validator", "error", err)
 			return false, err
 		}
-		return block_ref.Hash == latestBlock.Hash, nil
+		return blockRef.Hash == latestBlock.Hash, nil
 	})
 	require.NoError(t, err, "Expected block hash to match latest block hash on validator")
 

--- a/crates/optimism/tests/proofs/reorg/reorg_test.go
+++ b/crates/optimism/tests/proofs/reorg/reorg_test.go
@@ -174,7 +174,7 @@ func TestReorgUsingAccountProof(gt *testing.T) {
 		if err != nil {
 			// this could happen if the validator is still syncing after reorg
 			l.Warn("Error fetching block reference from validator", "error", err)
-			return false, err
+			return false, nil
 		}
 		return blockRef.Hash == latestBlock.Hash, nil
 	})

--- a/crates/optimism/tests/proofs/reorg/reorg_test.go
+++ b/crates/optimism/tests/proofs/reorg/reorg_test.go
@@ -6,6 +6,7 @@ import (
 	"time"
 
 	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/wait"
 	"github.com/ethereum-optimism/optimism/op-service/eth"
 	"github.com/ethereum-optimism/optimism/op-service/txplan"
 	"github.com/ethereum-optimism/optimism/op-test-sequencer/sequencer/seqtypes"
@@ -166,6 +167,18 @@ func TestReorgUsingAccountProof(gt *testing.T) {
 
 	latestBlock := sys.L2Chain.WaitForBlock()
 	sys.L2ELValidatorNode().WaitForBlockNumber(latestBlock.Number)
+
+	// verify that the L2A validator has reorged and reached the latest block
+	err := wait.For(t.Ctx(), 2*time.Second, func() (bool, error) {
+		block_ref, err := sys.L2ELValidatorNode().Escape().EthClient().BlockRefByNumber(ctx, latestBlock.Number)
+		if err != nil {
+			// this could happen if the validator is still syncing after reorg
+			l.Warn("Error fetching block reference from validator", "error", err)
+			return false, err
+		}
+		return block_ref.Hash == latestBlock.Hash, nil
+	})
+	require.NoError(t, err, "Expected block hash to match latest block hash on validator")
 
 	reorgedRef_A, err := sys.L2ELSequencerNode().Escape().EthClient().BlockRefByNumber(ctx, divergenceBlockNumber)
 	require.NoError(t, err, "Expected to be able to call BlockRefByNumber API, but got error")

